src/datos.json

```
@@ -0,0 +1,28 @@
+{
+    "contactInfo": {
+        "fullName": "Jose Eduardo Lopez Gonzalez",
+        "emailAddress": "jeduardo14lopez@gmail.com"
+    },
+    "github": {
+        "profileURL": "https://github.com/jelg14",
+        "username": "jelg14"
+    },
+    "credentials": {
+        "password": "asdf"
```

```suggestion
    "credentials": {
        "password": ""
```


Ingresar la contraseña directamente desde el código es una mala practica de seguridad. Y veo que al parecer tambien la tienes guardado en `process.env.PASSWORD`. Lo cual es correcto.

Por favor puedes borrar la contraseña del `.json`. Y realizar los cambios necesarios en tu código para que continué funcionando.

**Sugerencia**: Para evitar redundancias, mantenerlo modular, y no manipular mucho de los datos dentro del `client.ts`. Y ya que no se pueden asignar valores desde el `datos.json`, puedes utilizar un `datos.ts` para asignar los valores necesarios para el POST. He importar para utilizar en `client.ts`.